# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Define the goal
# Index with the valve has to be turnned
int32 valve_id
bool long_approach

